aap_compose_dev.yaml

```diff
@@ -88,24 +91,46 @@ services:
     volumes:
       - "etc_pulp_certs:/etc/pulp/certs"
       - "var_lib_pulp:/var/lib/pulp"
+      - ".:/src/galaxy_ng"
     command: |
       bash -c "
-        if [[ ! -e /etc/pulp/certs/database_fields.symmetric.key ]] || [[ -s /etc/pulp/certs/database_fields.symmetric.key ]]; then
-          mkdir -p /etc/pulp/certs/;
-          echo 'check openssl and install ...';
-          rpm -q openssl || dnf -y install openssl;
-          echo 'generate key ...';
-          openssl rand -base64 32 > /etc/pulp/certs/database_fields.symmetric.key;
-          echo 'chown key ...';
-          chmod 640 /etc/pulp/certs/database_fields.symmetric.key;
-        else
-          echo 'symmetric key exists'
-        fi;
+        echo '#> STEP: Database Symmetric Key';
+        echo 'WARNING: Symmetric key is hardcoded for development only.';
+        echo 'DNmNdwgyZugTax9S64J0FITTr9IHPxbuoF1F1CGPr68=' > /etc/pulp/certs/database_fields.symmetric.key;
         find /etc/pulp ;
         echo '# KEY >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>';
         cat /etc/pulp/certs/database_fields.symmetric.key;
         echo '<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<';
-        echo "DONE!";
+        echo 'DONE! <#';
+
+        echo '#> STEP: Signing scripts';
+        base64 -d <<< 'IyEvdXNyL2Jpbi9lbnYgYmFzaApHTlVQR0hPTUU9L3Jvb3QvLmdudXBnLwpncGcgLS1sb2NrLW5ldmVyIC0tcXVpZXQgLS1iYXRjaCAtLXBpbmVudHJ5LW1vZGUgbG9vcGJhY2sgLS15ZXMgLS1wYXNzcGhyYXNlIEdhbGF4eTIwMjQgLS1ob21lZGlyIC9yb290Ly5nbnVwZy8gLS1kZXRhY2gtc2lnbiAtLWRlZmF1bHQta2V5ICRQVUxQX1NJR05JTkdfS0VZX0ZJTkdFUlBSSU5UIC0tYXJtb3IgLS1vdXRwdXQgJDEuYXNjICQxClsgJD8gLWVxIDAgXSAmJiBlY2hvIHtcImZpbGVcIjogXCIkMVwiLCBcInNpZ25hdHVyZVwiOiBcIiQxLmFzY1wifSB8fCBleGl0ICQ/Cg==' > /var/lib/pulp/scripts/collection_sign.sh;
+        base64 -d <<< 'IyEvdXNyL2Jpbi9lbnYgYmFzaApza29wZW8gc3RhbmRhbG9uZS1zaWduIC0tcGFzc3BocmFzZS1maWxlIC9ldGMvcHVscC9jZXJ0cy9rZXlfcGFzc3dvcmQudHh0ICQxICRSRUZFUkVOQ0UgJFBVTFBfU0lHTklOR19LRVlfRklOR0VSUFJJTlQgLS1vdXRwdXQgJFNJR19QQVRIClsgJD8gLWVxIDAgXSAmJiBlY2hvIHtcInNpZ25hdHVyZV9wYXRoXCI6IFwiJFNJR19QQVRIXCJ9IHx8IGV4aXQgJD8=' > /var/lib/pulp/scripts/container_sign.sh;
+        chmod +x /var/lib/pulp/scripts/*_sign.sh;
+        ls -la /var/lib/pulp/scripts/collection_sign.sh;
+        cat /var/lib/pulp/scripts/collection_sign.sh;
+        echo '  ';
+        ls -la /var/lib/pulp/scripts/container_sign.sh;
+        cat /var/lib/pulp/scripts/container_sign.sh;
+        echo '  ';
+        echo 'DONE! <#';
```

Comment:
I made the scripts a little bit compact to save encoded chars

decoded:

collection

```bash
#!/usr/bin/env bash
GNUPGHOME=/root/.gnupg/
gpg --lock-never --quiet --batch --pinentry-mode loopback --yes --passphrase Galaxy2024 --homedir /root/.gnupg/ --detach-sign --default-key $PULP_SIGNING_KEY_FINGERPRINT --armor --output $1.asc $1
[ $? -eq 0 ] && echo {\"file\": \"$1\", \"signature\": \"$1.asc\"} || exit $?
```
container
```bash
#!/usr/bin/env bash
skopeo standalone-sign --passphrase-file /etc/pulp/certs/key_password.txt $1 $REFERENCE $PULP_SIGNING_KEY_FINGERPRINT --output $SIG_PATH
[ $? -eq 0 ] && echo {\"signature_path\": \"$SIG_PATH\"} || exit $?
```